account_analytic_salesperson/readme/ROADMAP.rst

```diff
@@ -0,0 +1,3 @@
+* As the module now includes two features (Creation of analytic account for
+  every sale order and addition of the salesperson field), this module could
+  be split into two different modules.
```

Comment:
I personally don't would make this solution. But of course I don't know the need. If you need that every sale order have analytic account just configure the product as create project, or just make a little customization to make the field analytic_account mandatory in the sale order :thinking:

Reply:
Sometimes we need to create a analytic account even the sales order doesn't contain a product type service. Hardware for example. I can't use the standard to create a project if I sell a physical product.